unit-tests/heitech.zer0mqXt.core.tests/PubSubTests.cs

```diff
@@ -124,7 +145,7 @@ public async Task Multiple_Subscriber_on_Single_Publisher()
         public class Message 
         {
             public string ThisIsAPublishedMessageText { get; set; }
-            public int[] Array { get; set; }
+            public int[] Array { get; set; } = Enumerable.Empty<int>().ToArray();
```

Comment:
Could be directly `Array.Empty<int>()` instead.

Reply:
Yes. You know me I am lazy, and this is from a snippet (:. So I guess I have to change this code and my snippet :D

Reply:
Ah, I see.
Also, I thought that this would be an unnecessary allocation and would anger the machine (in this case the _garbage collector_). Turns out that `ToArray()` actually calls `Array.Empty<T>()` when the count of the `IEnumerable<T>` is zero. Using `Array.Empty<int>()` directly is just a tiny little bit faster, nanoseconds-style.

Reply:
done